packages/react-components/react-table/src/components/TableCellLayout/TableCellLayout.types.ts

```diff
@@ -41,10 +41,18 @@ export type TableCellLayoutProps = ComponentProps<Partial<TableCellLayoutSlots>>
    * @default undefined
    */
   appearance?: 'primary';
+
+  /**
+   * Renders content with hidden overflow and ellipsis enabled
+   */
+  ellipsis?: boolean;
```

Comment:
`Text` component calls this prop `truncate`, let's be aligned

https://react.fluentui.dev/?path=/docs/components-text--default#truncate